sos/report/plugins/qaucli.py

```diff
@@ -0,0 +1,37 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, IndependentPlugin
+import os
+
+
+class Qaucli(Plugin, IndependentPlugin):
+
+    short_desc = 'QLogic information'
+
+    plugin_name = 'qaucli'
+    profiles = ('system', 'storage', 'hardware',)
+    packages = ('QConvergeConsoleCLI',)
+
+    def setup(self):
+        os.environ['PATH'] += os.pathsep + \
+            "/opt/QLogic_Corporation/QConvergeConsoleCLI"
```

Comment:
Nack. Do not manually modify the `PATH` for this, the sos API already accounts for standardized `PATH`s, and additionally doing things this way can interfere with non-standard `--sysroot`s.

Reply:
The CLI is `/opt/QLogic_Corporation/QConvergeConsoleCLI/qaucli`. What would be appropriate way to set the path?

Reply:
You can just use that full path as the command.

If you wanted to make the resulting filenames more human friendly, you'd need to change your command loop a bit to something like:

```python
cmd = "/opt/QLogic_Corporation/QConvergeConsoleCLI/qaucli"
for subcmd in subcmds:
    self.add_cmd_output("%s %s" % (cmd, subcmd),
                        suggest_filename="quacli_%s" % subcmd)
```

Reply:
That won't work for options (`subcmd`) with space. Is there a possibility to change the sosreport framework to
1. pick command basename when creating output file,
2. create command alias as a function parameter, or
3. update PATH env variable?

Reply:
I just tried your suggestion. It worked for the space too. Thank you @TurboTurtle .